src/common/activities/hackathon/HackathonBanner.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
+import { AiFillGithub } from "react-icons/ai";
+import { Link } from "react-router-dom";
+import { MdManageSearch } from "react-icons/md";
+import { FiStar } from "react-icons/fi";
+import HackerBannerImage from "./banner.png";
+import useFetch from "common/hooks/useFetch";
+import "./hackathonBanner.css";
+
+function HackathonBanner() {
+  const { data } = useFetch(
+    "https://api.github.com/repos/reactplay/react-play"
+  );
+  return (
+    <div className="flex flex-col lg:flex-row justify-between w-[98%]  max-w-[600px]  md:max-w-[992px] lg:max-w-[1200px] mt-16 m-auto text-white">
+      <div className="w-[90%] lg:w-[60%] mx-auto text-center lg:text-left">
+        <h1 className="inline-block text-4xl md:text-5xl lg:text-7xl font-bold">
+          ReactPlay
+        </h1>
+        <span className="text-lg md:text-xl lg:text-2xl">Presents</span>
+        <h1 className=" uppercase text-5xl md:text-6xl lg:text-8xl font-bold text-cyan-300">
+          Hackathon
+        </h1>
+        <span className="text-xl md:text-2xl lg:text-3xl text-cyan-300">
+          16 Sep - 8 Oct
+        </span>
+        <div className="my-2 md:my-4 md:text-xl">
+          <p>
+            Developers and Hacking are inseparable! ReactPlay brings you the
+            opportunity to take part in the Hackathon and learn from it.
+            Showcase your mindblowing ideas, build projects, and create content
+            - there are also chances to win exciting prizes
+          </p>
+        </div>
+        <div className="flex flex-col sm:flex-row justify-center lg:justify-start w-full ">
+          <button className="p-4 lg:w-48 my-2 rounded-full bg-cyan-300 font-bold  cursor-pointer">
```

Comment:
It should be a link, link to `https://hustles.reactplay.io/` and opens in the new tab. However the look and feel of it should be how it is now.